Magento2/Tests/Eslint/AbstractEslintTestCase.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento2\Tests\Eslint;
+
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractEslintTestCase extends TestCase
+{
+    protected function assertFileContainsError(string $testFile, array $expectedMessages)
```

Comment:
can you please add strict types and phpdocs to the added tests

Reply:
should be done, please check it out!